x-pack/plugins/lists/public/exceptions/hooks/use_exception_list.tsx

```diff
@@ -24,8 +32,7 @@ export type ReturnExceptionListAndItems = [boolean, ExceptionListAndItems | null
  */
 export const useExceptionList = ({
   http,
-  id,
-  namespaceType,
+  lists,
```

Comment:
nit: JSDoc will need updated with these changes. Since these are going to continue to change, feel free to throw a `TODO: ` above and we can just update them all in once fell swoop closer to FF. Whichever you prefer 🙂